src/components/NodesTable/NodesTable.vue

```diff
@@ -21,7 +21,21 @@ export default defineComponent({
   },
   setup() {
     const store = useStore()
-    const nodes = computed(() => store.getters['nodes/list'])
+    const nodes = computed(() => {
+      const arr = store.getters['nodes/list']
+      return arr.sort((a, b) => {
+        const alphabets = /^[a-zA-Z]+/;
+        const aAlphabets = a.url.replace(/\d+/g, "");
+        const bAlphabets = b.url.replace(/\d+/g, "");
+        if (aAlphabets === bAlphabets) {
+          const aNumber = a.url.replace(alphabets, "");
+          const bNumber = b.url.replace(alphabets, "");
+          const result = aNumber === bNumber ? 0 : parseInt(aNumber, 10) - parseInt(bNumber, 10);
+          return result;
+        }
+        return aAlphabets > bAlphabets ? 1 : -1;
+      })
+    })
```

Comment:
Overengineered. Simplify the sorting function. See examples on
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/sort

![image](https://github.com/Adamant-im/adamant-im/assets/25831507/405c6afe-4e3a-48f4-8347-e6420fbd5a61)